### PR TITLE
Adding logos to export modal

### PR DESCRIPTION
**Pull request by karanvirsb**
#227 

image
image

### CODE REVIEW

**Comment by alexdanilowicz** [comment]
@karanvirsb 
image

 if you could just update the language and move the logos to the below the code snippet that would be awesome! as discussed here: https://mirrorful.slack.com/archives/C04RR2D7C65/p1680567571567519

**Comment by alexdanilowicz** [comment]
Feel free to un mark it as draft when you think it's ready.

**Comment by Pranav2612000** [comment]
@alexdanilowicz I've added a review comment I thought would make the code better. I hope you don't mind.

**Comment by Pranav2612000** [comment]
Also @karanvirsb I'm sure you already plan to take care of it, but there are some unused imports like `Heading` from chakra-ui. Maybe we can remove them if we don't need the Components?

**Comment by karanvirsb** [comment]
> @karanvirsb image
> 
> if you could just update the language and move the logos to the below the code snippet that would be awesome! as discussed here: https://mirrorful.slack.com/archives/C04RR2D7C65/p1680567571567519

Yes I will work on this tomorrow as I was busy with the next js routing today.

File: packages/core/src/components/ExportSuccessModal.tsx [hunk 1]
```diff
@@ -389,6 +397,127 @@ function WebModalBody({ tokens }: { tokens: TTokens }) {
   )
 }
 
+function ExternalExamples() {
+  return (
+    <Flex css={{ marginTop: '8px' }} gap={8} flexWrap="wrap">
+      <Link
```
[the remark this answers is not in the document]
Comment (karanvirsb) [reply packages/core/src/components/ExportSuccessModal.tsx:403]: I totally agree was going to fix it! Thanks for the suggestion along with the Heading I will remove any unused imports at the end!

**Comment by alexdanilowicz** [comment]
Screen Shot 2023-04-05 at 12 01 00 PM
Screen Shot 2023-04-05 at 12 02 29 PM

**Comment by alexdanilowicz** [comment]
@karanvirsb cleaned up a few things ^ see above. just merged. well done!

**Comment by karanvirsb** [comment]
> @karanvirsb cleaned up a few things ^ see above. just merged. well done!

Looks awesome!

**Pull request merged**